README.md

```diff
@@ -81,6 +81,14 @@ Profiling using eBPF incurs lower overhead & provides kernel stacks. This (curre
 **By default, gProfiler sends logs to Granulate Performance Studio** (when using `--upload-results`/`-u` flag)
 This behavior can be disabled by passing `--dont-send-logs` or the setting environment variable `GPROFILER_DONT_SEND_LOGS=1`.
 
+### Metrics and metadata collection
+By default, the gProfiler agent sends system metrics (CPU and RAM usage) and metadata to the Performance Studio.
+The metadata includes system metadata like the kernel version and CPU count, and cloud metadata like the type of the instance you are running on.
+If the `--upload-results`/`-u` is not enabled, then the metrics and metadata collection is disabled.
```

Comment:
Instead of negative negative, explain it positive positive: metrics & metadata are sent only iff `-u`.